api/apps/api/src/modules/projects/dto/planning-area-created.dto.ts

```diff
@@ -0,0 +1,14 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { GeoJSON } from 'geojson';
+
+export class PlanningAreaCreatedDto {
+  @ApiProperty({
+    description: 'An ID of the created planning area',
+  })
+  id!: string;
+
+  @ApiProperty({
+    description: 'GeoJSON of the created planning area',
+  })
+  data!: GeoJSON;
+}
```

Comment:
very minor and subtle note - we already have a `ShapefileGeoJSONResponseDTO` which may partly overlap with this at least in intention, though that's a generic GeoJSON response DTO. Can we keep the two linked (e.g. by extending the generic one)?

Also, is this DTO unique to "created"-type responses? or is it a generic planning area result DTO, even if only used in "created"-type responses? If so, I would stick to something like `PlanningAreaResponseDTO` or something like this.

Reply:
ed1f455678144f55728ae940863249a58ca51cfa